ios/brave-ios/Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -108,21 +110,34 @@ struct SubmitReportView: View {
 
   @MainActor func createAndSubmitReport() async {
     let domain = Domain.getOrCreate(forUrl: url, persistent: !isPrivateBrowsing)
-
-    let report = WebcompatReporter.Report(
-      cleanedURL: url,
-      additionalDetails: additionalDetails,
-      contactInfo: contactDetails,
-      areShieldsEnabled: !domain.areAllShieldsOff,
-      adBlockLevel: domain.globalBlockAdsAndTrackingLevel,
-      fingerprintProtectionLevel: domain.finterprintProtectionLevel,
-      adBlockListTitles: FilterListStorage.shared.filterLists.compactMap({
-        return $0.isEnabled ? $0.entry.title : nil
-      }),
-      isVPNEnabled: BraveVPN.isConnected
+    let webcompatReporterAPI = WebcompatReporter.ServiceFactory.get(
+      privateMode: false
+    )!
```

Comment:
```suggestion
    guard
      let webcompatReporterAPI = WebcompatReporter.ServiceFactory.get(
        privateMode: isPrivateBrowsing
      )
    else {
      return
    }
```

Should avoid force unwrap here, and can pass in `isPrivateBrowsing` value for `privateMode` param